WordPress/Classes/ViewRelated/NUX/Helpers/WordPressAuthenticationManager.swift

```diff
@@ -213,7 +213,7 @@ extension WordPressAuthenticationManager {
     /// Presents the WordPress Authentication UI from the rootViewController (configured to allow only WordPress.com).
     /// This method pre-populates the Email + Username with the values returned by the default WordPress.com account (if any).
     ///
-    static func showSigninForWPComFixingAuthToken() {
+    static func showSigninForWPComFixingAuthToken(showNotice: Bool = true) {
```

Comment:
(nit) the new `showNotice` parameter appears to be unused. I'm not sure if you plan to use I somewhere?

Reply:
@kean I commented below but forgot to tag you: https://github.com/wordpress-mobile/WordPress-iOS/pull/24434#discussion_r2036234248